code/datums/ai/_ai_controller.dm

```diff
@@ -171,6 +171,15 @@ multiple modular subtrees with behaviors
 
 ///Runs any actions that are currently running
 /datum/ai_controller/process(seconds_per_tick)
+
+	var/mob/living/living_pawn = src.pawn
+
+	if(!length(living_pawn.do_afters))
+		living_pawn.ai_controller.set_blackboard_key(BB_RESISTING, FALSE)
+
+	if(living_pawn.ai_controller.blackboard[BB_RESISTING])
+		return
```

Comment:
I don't think we should be checking and resetting this key on every AI when only one AI behaviour ever sets it. Is there no other way to get this result?

Reply:
I could move it to the general ai_behavior datum checks so that if it requires movement it'll check if they're not currently resisting, but I'm not too sure where to put it to guarantee that they'll check if they're resisting. Alternatively I can just put it in a moderately common behavior so that it isn't ticking for it constantly, but they'll be a bit slower to react to resisting, but it's better than nothing.

Reply:
I could also just make it so monkeys are the only ai that constantly checks to resist, I don't really see any other ai that resists and moving the resisting check to in between behaviors means they can only break cuffs and not resist out of grabs.